contracts/SecurityTokenRegistry.sol

```diff
@@ -670,6 +679,25 @@ contract SecurityTokenRegistry is EternalStorage, Proxy {
         set(STLAUNCHFEE, _stLaunchFee);
     }
 
+    /**
+    * @notice Sets the ticker registration and ST launch fee amount and currency
+    * @param _tickerRegFee is the ticker registration fee (base 18 decimals)
+    * @param _stLaunchFee is the st generation fee (base 18 decimals)
+    * @param _isFeeInPoly defines if the fee is in poly or usd
+    */
+    function changeFeesAmountAndCurrency(uint256 _tickerRegFee, uint256 _stLaunchFee, bool _isFeeInPoly) external onlyOwner {
+        uint256 tickerFee = getUintValue(TICKERREGFEE);
+        uint256 stFee = getUintValue(STLAUNCHFEE);
+        bool oldIsFeesInPoly = getBoolValue(IS_FEE_IN_POLY);
+        require(oldIsFeesInPoly != _isFeeInPoly, "Currency unchanged");
+        emit ChangeTickerRegistrationFee(tickerFee, _tickerRegFee);
```

Comment:
better to use _changeTickerRegistrationFee() and _changeSecurityLaunchFee() function here

Reply:
done